wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -152,18 +152,16 @@ class SwiftGenerator private constructor(
     get() = jsonName?.takeIf { it != name } ?: camelCase(name).takeIf { it != name }
 
   private val Field.typeName: TypeName
-    get() {
-      return when (encodeMode!!) {
-        EncodeMode.MAP -> DICTIONARY.parameterizedBy(keyType.typeName, valueType.typeName)
-        EncodeMode.REPEATED,
-        EncodeMode.PACKED -> ARRAY.parameterizedBy(type!!.typeName)
-        EncodeMode.NULL_IF_ABSENT -> OPTIONAL.parameterizedBy(type!!.typeName)
-        EncodeMode.REQUIRED -> type!!.typeName
-        EncodeMode.OMIT_IDENTITY -> {
-          when {
-            isOneOf || isMessage -> OPTIONAL.parameterizedBy(type!!.typeName)
-            else -> type!!.typeName
-          }
+    get() = when (encodeMode!!) {
```

Comment:
Consistency change of using `get() =`; otherwise unchanged.
I made this change while trying to make the `Swift.` stuff go away.